index.html

```diff
@@ -0,0 +1,174 @@
+				<ul class="description">
+					<li><h3>FACEBOOK</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Full Stack Dev</li>
+					<li><i class="fa-solid fa-circle"></i> 2015</li>
+				</ul>
+				<p>
+					Experimental content creation feature that allows users to
+					add to an existing story over the course of a day without
+					spamming without their friends.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+
+			<div class="card">
+				<img src="images/Portfolio-3.png" alt="" />
+				<h1>Tonic</h1>
+				<ul class="description">
+					<li><h3>CANOPY</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Back End Dev</li>
+					<li><i class="fa-solid fa-circle"></i> 2018</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no
+					accounts or sign-ups required.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+
+			<div class="card">
+				<img src="images/Portfolio-4.png" alt="" />
+				<h1>Uber Navigation</h1>
+				<ul class="description">
+					<li><h3>Uber</h3></li>
+					<li><i class="fa-solid fa-circle"></i> Lead developer</li>
+					<li><i class="fa-solid fa-circle"></i> 2018</li>
+				</ul>
+				<p>
+					A smart assistant to make driving more safe, efficient, and
+					fun by unlocking your most expensive computer: your car.
+				</p>
+				<ul class="language-used">
+					<li>html</li>
+					<li>css</li>
+					<li>javaScript</li>
+				</ul>
+				<button type="submit">See Project</button>
+			</div>
+		</section>
+
+		<section>
+			<div class="about">
+				<h1>About Myself</h1>
+				<p>
+					Hello I'm a software developer! I can help you build a
+					product, feature or website Look through some of my work and
+					experience! If you like what you see and have a project you
+					need coded, don't hesitate to contact me.
+				</p>
+				<a href="connect">LET'S CONNECT</a>
+
+				<ul class="social-links">
+					<li class="social-link-item">
+						<a href="https://twitter.com/AddisuTedla"
+							><i class="fa-brands fa-twitter"></i
+						></a>
+					</li>
+					<li class="social-link-item">
+						<a
+							href="https://www.linkedin.com/in/addisu-tedla-8b4a10143/"
+							><i class="fa-brands fa-linkedin"></i
+						></a>
+					</li>
+					<li class="social-link-item">
+						<a href="https://github.com/Addisu87"
+							><i class="fa-brands fa-github"></i
+						></a>
+					</li>
+					<li class="social-link-item">
+						<a href="https://medium.com/@addisuhaile87"
+							><i class="fa-brands fa-medium"></i
+						></a>
+					</li>
+					<li class="social-link-item">
+						<a href=""><i class="fa-brands fa-angellist"></i></a>
+					</li>
+				</ul>
+
+				<button class="resume" type="submit">Get my resume</button>
```

Comment:
- [ ] The button ( "Get my resume") should follow the button styling: Active, Hover and Pressed ( see screenshot below);
![get my Resume](https://user-images.githubusercontent.com/65852621/164825993-92853639-5edb-46d4-8caa-a19d7657e604.png)